<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>GSE19711</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Cancer Effects in Blood</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="GSE19711.html">GSE19711</a>
</li>
<li>
  <a href="GSE30229.html">GSE30229</a>
</li>
<li>
  <a href="GSE50409.html">GSE50409</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">GSE19711</h1>

</div>


<section id="authors" class="level1">
<h1>Authors</h1>
<ul>
<li>Rimvydas Noreika</li>
<li>Tautminas Cibulskis</li>
<li>Gabrielius Erignis</li>
<li>Rolandas PorÄ—jus</li>
</ul>
</section>
<section id="analysis" class="level1">
<h1>Analysis</h1>
<ul>
<li>Automatically download the data from GEO</li>
</ul>
<pre class="r"><code>gset = getGEO(&quot;GSE19711&quot;, destdir=&quot;./&quot;)</code></pre>
<pre><code>## Found 1 file(s)</code></pre>
<pre><code>## GSE19711_series_matrix.txt.gz</code></pre>
<pre><code>## Using locally cached version: .//GSE19711_series_matrix.txt.gz</code></pre>
<pre><code>## Parsed with column specification:
## cols(
##   .default = col_double(),
##   ID_REF = col_character()
## )</code></pre>
<pre><code>## See spec(...) for full column specifications.</code></pre>
<pre><code>## Using locally cached version of GPL8490 found here:
## .//GPL8490.soft</code></pre>
<ul>
<li>Obtain the matrix of beta values where each row corresponds to probes and each column corresponds to samples</li>
</ul>
<pre class="r"><code>gsetexpr = exprs(gset[[1]])
rownames(gsetexpr)[1:5]</code></pre>
<pre><code>## [1] &quot;cg00000292&quot; &quot;cg00002426&quot; &quot;cg00003994&quot; &quot;cg00005847&quot; &quot;cg00006414&quot;</code></pre>
<pre class="r"><code>colnames(gsetexpr)[1:5]</code></pre>
<pre><code>## [1] &quot;GSM491937&quot; &quot;GSM491938&quot; &quot;GSM491939&quot; &quot;GSM491940&quot; &quot;GSM491941&quot;</code></pre>
<ul>
<li>How many samples and how many probes do you have in the data?</li>
</ul>
<pre class="r"><code>sampleset = colnames(gsetexpr)
length(samples)</code></pre>
<pre><code>## [1] 1</code></pre>
<pre class="r"><code>probeset = rownames(gsetexpr)
length(probeset)</code></pre>
<pre><code>## [1] 27578</code></pre>
<ul>
<li>How are the beta values distributed?</li>
</ul>
<pre class="r"><code>hist(gsetexpr)</code></pre>
<p><img src="GSE19711_files/figure-html/unnamed-chunk-5-1.png" width="672" /> - Do your probes have names?</p>
<pre class="r"><code>probeset[1:5]</code></pre>
<pre><code>## [1] &quot;cg00000292&quot; &quot;cg00002426&quot; &quot;cg00003994&quot; &quot;cg00005847&quot; &quot;cg00006414&quot;</code></pre>
<ul>
<li>Do you have annotation that tells the coordinate (in hg19) of each probe and its genomic features (such as related gene name)?</li>
</ul>
<pre class="r"><code>annotation = getGEO(&quot;GPL8490&quot;, destdir = &quot;./&quot;)</code></pre>
<pre><code>## Using locally cached version of GPL8490 found here:
## .//GPL8490.soft</code></pre>
<pre class="r"><code>annotation = Table(annotation)
head(annotation)</code></pre>
<pre><code>##           ID       Name IlmnStrand AddressA_ID
## 1 cg00000292 cg00000292        TOP      990370
## 2 cg00002426 cg00002426        TOP     6580397
## 3 cg00003994 cg00003994        TOP     7150184
##                                     AlleleA_ProbeSeq AddressB_ID
## 1 AAACATTAATTACCAACCACTCTTCCAAAAAACACTTACCATTAAAACCA     6660678
## 2 AATATAATAACATTACCTTACCCATCTTATAATCAAACCAAACAAAAACA     6100343
## 3 AATAATAATAATACCCCCTATAATACTAACTAACAAACATACCCTCTTCA     7150392
##                                     AlleleB_ProbeSeq GenomeBuild Chr
## 1 AAACATTAATTACCAACCGCTCTTCCAAAAAACACTTACCATTAAAACCG          36  16
## 2 AATATAATAACATTACCTTACCCGTCTTATAATCAAACCAAACGAAAACG          36   3
## 3 AATAATAATAATACCCCCTATAATACTAACTAACAAACATACCCTCTTCG          36   7
##     MapInfo  Ploidy      Species      Source SourceVersion SourceStrand
## 1  28797601 diploid Homo sapiens NCBI:RefSeq          36.1          TOP
## 2  57718583 diploid Homo sapiens NCBI:RefSeq          36.1          TOP
## 3  15692387 diploid Homo sapiens NCBI:RefSeq          36.1          BOT
##                                            SourceSeq
## 1 CGGCCTCAATGGTAAGTGTCCCTTGGAAGAGCGGCTGGTAATTAATGCCC
## 2 CGCTCTCGTCTGGTTTGATCACAAGACGGGCAAGGTAATGTCACCACATT
## 3 GGTGGTGGTGGTGCCCCCTGTGATGCTGGCTGGCAAACATGCCCTCTTCG
##                                                                                                                  TopGenomicSeq
## 1 TGGGGTGAGTGAGACCACGGGCCTCACCCCGGACCAAGTTAAGCGGAATCTGGAGAAATA[CG]GCCTCAATGGTAAGTGTCCCTTGGAAGAGCGGCTGGTAATTAATGCCCTCCTGCACCCCC
## 2 CCGCTGTCGACCAGCGCAGAATAATGCCACTTTTGATTGCAAAGTGCTATCAAGGAACCA[CG]CTCTCGTCTGGTTTGATCACAAGACGGGCAAGGTAATGTCACCACATTGTCCAGCGGCAT
## 3 GGTGGTGGTGGTGGTGGTGGTGGTGCCCCCTGTGATGCTGGCTGGCAAACATGCCCTCTT[CG]TTGGGGTATCCCGCGATTATGCAAGATGAGGAAGAAGTAGAGAGCTCGGGGTAAGACATA
##   Next_Base Color_Channel TSS_Coordinate Gene_Strand      Gene_ID Symbol
## 1         T           Red       28797310           +   GeneID:487 ATP2A1
## 2         T           Red       57718214           +  GeneID:7871  SLMAP
## 3         T           Red       15692819           -  GeneID:4223  MEOX2
##                                  Synonym   Accession         GID
## 1                         ATP2A; SERCA1; NM_173201.2 GI:47132613
## 2                        SLAP; KIAA1601; NM_007159.2 GI:56550042
## 3                             GAX; MOX2; NM_005924.3 GI:55956906
##                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         Annotation
## 1 isoform a is encoded by transcript variant a; sarcoplasmic/endoplasmic reticulum calcium ATPase 1; calcium pump 1; SR Ca(2+)-ATPase 1; calcium-transporting ATPase sarcoplasmic reticulum type; fast twitch skeletal muscle isoform; endoplasmic reticulum class 1/2 Ca(2+) ATPase; go_component: membrane; go_component: integral to membrane; go_component: sarcoplasmic reticulum; go_component: smooth endoplasmic reticulum; go_function: ATP binding; go_function: hydrolase activity; go_function: nucleotide binding; go_function: calcium ion binding; go_function: magnesium ion binding; go_function: calcium-transporting ATPase activity; go_function: hydrolase activity; acting on acid anhydrides; catalyzing transmembrane movement of substances; go_process: metabolism; go_process: cation transport; go_process: proton transport; go_process: calcium ion transport; go_process: regulation of striated muscle contraction
## 2                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        Sarcolemmal-associated protein; go_component: integral to plasma membrane; go_component: smooth endoplasmic reticulum; go_function: unfolded protein binding; go_process: protein folding; go_process: muscle contraction
## 3                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    growth arrest-specific homeo box; go_component: nucleus; go_function: transcription factor activity; go_process: circulation; go_process: development; go_process: regulation of transcription; DNA-dependent
##                                              Product Distance_to_TSS
## 1 ATPase; Ca++ transporting; fast twitch 1 isoform a             291
## 2                      sarcolemma associated protein             369
## 3                             mesenchyme homeo box 2             432
##   CPG_ISLAND  CPG_ISLAND_LOCATIONS MIR_CPG_ISLAND     RANGE_GB RANGE_START
## 1       TRUE  16:28797486-28797825                 NC_000016.8    28797486
## 2       TRUE   3:57716811-57718675                NC_000003.10    57716811
## 3       TRUE   7:15691512-15693551                NC_000007.12    15691512
##   RANGE_END RANGE_STRAND      GB_ACC   ORF
## 1  28797825            + NM_173201.2   487
## 2  57718675            + NM_007159.2  7871
## 3  15693551            - NM_005924.3  4223
##  [ reached getOption(&quot;max.print&quot;) -- omitted 3 rows ]</code></pre>
<ul>
<li>Do you know which samples correspond to healthy individuals, and which samples correspond to the sick ones?</li>
</ul>
<pre class="r"><code>disease = pData(phenoData(gset[[1]]))
disease[1:20,1]</code></pre>
<pre><code>##  [1] Peripheral Whole Blood 109164 Peripheral Whole Blood 118275
##  [3] Peripheral Whole Blood 113964 Peripheral Whole Blood 109567
##  [5] Peripheral Whole Blood 100793 Peripheral Whole Blood 115541
##  [7] Peripheral Whole Blood 111221 Peripheral Whole Blood 111133
##  [9] Peripheral Whole Blood 111871 Peripheral Whole Blood 111572
## [11] Peripheral Whole Blood 111164 Peripheral Whole Blood 111540
## [13] Peripheral Whole Blood 109645 Peripheral Whole Blood 116007
## [15] Peripheral Whole Blood 113005 Peripheral Whole Blood 104583
## [17] Peripheral Whole Blood 114766 Peripheral Whole Blood 109691
## [19] Peripheral Whole Blood 119821 Peripheral Whole Blood 113049
## 540 Levels: Peripheral Whole Blood 100097 ... Peripheral Whole Blood 123698</code></pre>
</section>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
