<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>GSE30229</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Cancer Effects in Blood</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="GSE19711.html">GSE19711</a>
</li>
<li>
  <a href="GSE30229.html">GSE30229</a>
</li>
<li>
  <a href="GSE50409.html">GSE50409</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">GSE30229</h1>
<h4 class="date"><em>2018 m. kovas</em></h4>

</div>


<section id="authors" class="level1">
<h1>Authors</h1>
<ul>
<li>Edgaras Legus</li>
<li>Vitalija Misiukonytė</li>
<li>Brigita Ižganaitytė</li>
<li>Gabrielė Dilytė</li>
<li>Dalia Masilionytė</li>
<li>Reda Vaisėtaitė</li>
<li>Paulius Matijošaitis</li>
<li>Dovilė Patiejūnaitė</li>
</ul>
</section>
<section id="analysis" class="level1">
<h1>Analysis</h1>
<section id="step-2" class="level2">
<h2>Step 2</h2>
<p>Downloading the data using GEO</p>
<pre class="r"><code>library(GEOquery)
GSE30229 &lt;- getGEO(&quot;GSE30229&quot;, destdir = &quot;./&quot;)</code></pre>
<pre><code>## Found 1 file(s)</code></pre>
<pre><code>## GSE30229_series_matrix.txt.gz</code></pre>
<pre><code>## Using locally cached version: .//GSE30229_series_matrix.txt.gz</code></pre>
<pre><code>## Parsed with column specification:
## cols(
##   .default = col_double(),
##   ID_REF = col_character()
## )</code></pre>
<pre><code>## See spec(...) for full column specifications.</code></pre>
<pre><code>## Using locally cached version of GPL8490 found here:
## .//GPL8490.soft</code></pre>
<p>Obtaining the betaValueMatrix of beta values where each row corresponds to probes (names) and each column corresponds to samples</p>
<pre class="r"><code>betaValueMatrix &lt;- exprs(GSE30229[[1]])
head(rownames(betaValueMatrix))</code></pre>
<pre><code>## [1] &quot;cg00000292&quot; &quot;cg00002426&quot; &quot;cg00003994&quot; &quot;cg00005847&quot; &quot;cg00006414&quot;
## [6] &quot;cg00007981&quot;</code></pre>
<pre class="r"><code>head(colnames(betaValueMatrix))</code></pre>
<pre><code>## [1] &quot;GSM748577&quot; &quot;GSM748578&quot; &quot;GSM748579&quot; &quot;GSM748580&quot; &quot;GSM748581&quot; &quot;GSM748582&quot;</code></pre>
<p>Counting samples and probes in our data</p>
<pre class="r"><code>samples &lt;- colnames(betaValueMatrix)
probes &lt;- rownames(betaValueMatrix)
length(samples)</code></pre>
<pre><code>## [1] 184</code></pre>
<pre class="r"><code>length(probes)</code></pre>
<pre><code>## [1] 27578</code></pre>
<p>Distribution of beta values</p>
<pre class="r"><code>hist(betaValueMatrix, breaks=1000, xlab = &quot;Beta Value &quot;, border = &quot;green&quot;, main = &quot;Beta value distribution&quot;)</code></pre>
<p><img src="GSE30229_files/figure-html/unnamed-chunk-4-1.png" width="672" /></p>
<p>Names of probes</p>
<pre class="r"><code>head(rownames(betaValueMatrix))</code></pre>
<pre><code>## [1] &quot;cg00000292&quot; &quot;cg00002426&quot; &quot;cg00003994&quot; &quot;cg00005847&quot; &quot;cg00006414&quot;
## [6] &quot;cg00007981&quot;</code></pre>
<p>Annotation that tells the coordinate (in hg19) of each probe and its genomic features</p>
<pre class="r"><code>annotate &lt;- getGEO(&quot;GPL8490&quot;, destdir = &quot;./&quot;)</code></pre>
<pre><code>## Using locally cached version of GPL8490 found here:
## .//GPL8490.soft</code></pre>
<pre class="r"><code>annotate &lt;- Table(annotate)

annotated_probes &lt;- intersect(annotate$ID, rownames(betaValueMatrix))
cat(&quot;Probes number: &quot;, length(annotated_probes), &quot;\n&quot;)</code></pre>
<pre><code>## Probes number:  27578</code></pre>
<pre class="r"><code>i &lt;- match(annotated_probes, annotate$ID)
annotate &lt;- annotate[i, ]

i &lt;- match(annotated_probes, rownames(betaValueMatrix))
betaValueMatrix &lt;- betaValueMatrix[i,]
stopifnot(all(rownames(betaValueMatrix) == annotate$ID))

head(annotate)</code></pre>
<pre><code>##           ID       Name IlmnStrand AddressA_ID
## 1 cg00000292 cg00000292        TOP      990370
## 2 cg00002426 cg00002426        TOP     6580397
## 3 cg00003994 cg00003994        TOP     7150184
##                                     AlleleA_ProbeSeq AddressB_ID
## 1 AAACATTAATTACCAACCACTCTTCCAAAAAACACTTACCATTAAAACCA     6660678
## 2 AATATAATAACATTACCTTACCCATCTTATAATCAAACCAAACAAAAACA     6100343
## 3 AATAATAATAATACCCCCTATAATACTAACTAACAAACATACCCTCTTCA     7150392
##                                     AlleleB_ProbeSeq GenomeBuild Chr
## 1 AAACATTAATTACCAACCGCTCTTCCAAAAAACACTTACCATTAAAACCG          36  16
## 2 AATATAATAACATTACCTTACCCGTCTTATAATCAAACCAAACGAAAACG          36   3
## 3 AATAATAATAATACCCCCTATAATACTAACTAACAAACATACCCTCTTCG          36   7
##     MapInfo  Ploidy      Species      Source SourceVersion SourceStrand
## 1  28797601 diploid Homo sapiens NCBI:RefSeq          36.1          TOP
## 2  57718583 diploid Homo sapiens NCBI:RefSeq          36.1          TOP
## 3  15692387 diploid Homo sapiens NCBI:RefSeq          36.1          BOT
##                                            SourceSeq
## 1 CGGCCTCAATGGTAAGTGTCCCTTGGAAGAGCGGCTGGTAATTAATGCCC
## 2 CGCTCTCGTCTGGTTTGATCACAAGACGGGCAAGGTAATGTCACCACATT
## 3 GGTGGTGGTGGTGCCCCCTGTGATGCTGGCTGGCAAACATGCCCTCTTCG
##                                                                                                                  TopGenomicSeq
## 1 TGGGGTGAGTGAGACCACGGGCCTCACCCCGGACCAAGTTAAGCGGAATCTGGAGAAATA[CG]GCCTCAATGGTAAGTGTCCCTTGGAAGAGCGGCTGGTAATTAATGCCCTCCTGCACCCCC
## 2 CCGCTGTCGACCAGCGCAGAATAATGCCACTTTTGATTGCAAAGTGCTATCAAGGAACCA[CG]CTCTCGTCTGGTTTGATCACAAGACGGGCAAGGTAATGTCACCACATTGTCCAGCGGCAT
## 3 GGTGGTGGTGGTGGTGGTGGTGGTGCCCCCTGTGATGCTGGCTGGCAAACATGCCCTCTT[CG]TTGGGGTATCCCGCGATTATGCAAGATGAGGAAGAAGTAGAGAGCTCGGGGTAAGACATA
##   Next_Base Color_Channel TSS_Coordinate Gene_Strand      Gene_ID Symbol
## 1         T           Red       28797310           +   GeneID:487 ATP2A1
## 2         T           Red       57718214           +  GeneID:7871  SLMAP
## 3         T           Red       15692819           -  GeneID:4223  MEOX2
##                                  Synonym   Accession         GID
## 1                         ATP2A; SERCA1; NM_173201.2 GI:47132613
## 2                        SLAP; KIAA1601; NM_007159.2 GI:56550042
## 3                             GAX; MOX2; NM_005924.3 GI:55956906
##                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         Annotation
## 1 isoform a is encoded by transcript variant a; sarcoplasmic/endoplasmic reticulum calcium ATPase 1; calcium pump 1; SR Ca(2+)-ATPase 1; calcium-transporting ATPase sarcoplasmic reticulum type; fast twitch skeletal muscle isoform; endoplasmic reticulum class 1/2 Ca(2+) ATPase; go_component: membrane; go_component: integral to membrane; go_component: sarcoplasmic reticulum; go_component: smooth endoplasmic reticulum; go_function: ATP binding; go_function: hydrolase activity; go_function: nucleotide binding; go_function: calcium ion binding; go_function: magnesium ion binding; go_function: calcium-transporting ATPase activity; go_function: hydrolase activity; acting on acid anhydrides; catalyzing transmembrane movement of substances; go_process: metabolism; go_process: cation transport; go_process: proton transport; go_process: calcium ion transport; go_process: regulation of striated muscle contraction
## 2                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        Sarcolemmal-associated protein; go_component: integral to plasma membrane; go_component: smooth endoplasmic reticulum; go_function: unfolded protein binding; go_process: protein folding; go_process: muscle contraction
## 3                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    growth arrest-specific homeo box; go_component: nucleus; go_function: transcription factor activity; go_process: circulation; go_process: development; go_process: regulation of transcription; DNA-dependent
##                                              Product Distance_to_TSS
## 1 ATPase; Ca++ transporting; fast twitch 1 isoform a             291
## 2                      sarcolemma associated protein             369
## 3                             mesenchyme homeo box 2             432
##   CPG_ISLAND  CPG_ISLAND_LOCATIONS MIR_CPG_ISLAND     RANGE_GB RANGE_START
## 1       TRUE  16:28797486-28797825                 NC_000016.8    28797486
## 2       TRUE   3:57716811-57718675                NC_000003.10    57716811
## 3       TRUE   7:15691512-15693551                NC_000007.12    15691512
##   RANGE_END RANGE_STRAND      GB_ACC   ORF
## 1  28797825            + NM_173201.2   487
## 2  57718675            + NM_007159.2  7871
## 3  15693551            - NM_005924.3  4223
##  [ reached getOption(&quot;max.print&quot;) -- omitted 3 rows ]</code></pre>
<p>Samples which correspond to healthy individuals, and which samples correspond to the sick ones</p>
<pre class="r"><code>sickness &lt;- pData(phenoData(GSE30229[[1]]))
sickness[1:20,1]</code></pre>
<pre><code>##  [1] HNSCC case PBL 1  HNSCC case PBL 2  HNSCC case PBL 3 
##  [4] control PBL 4     control PBL 5     control PBL 6    
##  [7] control PBL 7     HNSCC case PBL 8  HNSCC case PBL 9 
## [10] control PBL 10    control PBL 11    HNSCC case PBL 12
## [13] HNSCC case PBL 13 HNSCC case PBL 14 control PBL 15   
## [16] control PBL 16    HNSCC case PBL 17 control PBL 18   
## [19] control PBL 19    HNSCC case PBL 20
## 184 Levels: control PBL 10 control PBL 101 ... HNSCC case PBL 99</code></pre>
</section>
<section id="step-3" class="level2">
<h2>Step 3</h2>
<p>For each probe compute a t-test to verify if the distributions of beta values within the probe significantly differ between the two groups.</p>
<pre class="r"><code>controlSamples &lt;- which(sickness$`disease:ch1` == &#39;control&#39;)
caseSamples &lt;- which(sickness$`disease:ch1` == &#39;HNSCC case&#39;)
t.test(betaValueMatrix[1,controlSamples], betaValueMatrix[1,caseSamples])</code></pre>
<pre><code>## 
##  Welch Two Sample t-test
## 
## data:  betaValueMatrix[1, controlSamples] and betaValueMatrix[1, caseSamples]
## t = 1.4715, df = 173.5, p-value = 0.143
## alternative hypothesis: true difference in means is not equal to 0
## 95 percent confidence interval:
##  -0.003265838  0.022401760
## sample estimates:
## mean of x mean of y 
## 0.7852802 0.7757123</code></pre>
<p>From the t-test, obtain the p value.</p>
<pre class="r"><code>pvalues = c()
for (i in 1:length(rownames(betaValueMatrix))) {
     pvalues[i] &lt;- t.test(betaValueMatrix[i,controlSamples], betaValueMatrix[i,caseSamples])$p.value
     
}</code></pre>
<p>Plot the distribution of p values. What is the expected distribution? How dows it differ from what you get?</p>
<pre class="r"><code>#The peak close to 0 is tall, so there are many p-values close to 0 which means that there is a lot of significant values.
#The “depth” of the histogram on the right side shows the values that are null.
hist(pvalues, col = &quot;green&quot;, breaks = 100)</code></pre>
<p><img src="GSE30229_files/figure-html/unnamed-chunk-10-1.png" width="672" /></p>
<p>Performance-wise, how long will it take to compute the test for all probes?</p>
<pre class="r"><code># We used system.time() function to measure the performance. It took 30.100 seconds to perform all the t-tests</code></pre>
</section>
<section id="step-4" class="level2">
<h2>Step 4</h2>
<p>What is multiple hypothesis testing?</p>
<pre class="r"><code>#Kitaip jie dar gali būti vadinami “p-hacking”. Tai statistinių metodų naudojimas tam, kad išgauti geresnius, reikšmingesnius rezultatus. 
#Yra eksperimentų, kurių duomenyse yra didelis kintamųjų skaičius. Tokie didelio našumo eksperimetai atliekami genetikoje ir image analysis. 

#Į šį testavimo būdą įeina:
#1) skirtumai tarp nulinės ir alternatyvios hipotezės
#2) reikšmingumo lygiai
#3) Type I and Type II errors
#4)p-values

#Atliekant multiple hypothesis testing mes tikriname individualiai kiekvieną nulinę hipotezę. Kad galėtume atmesti bent kelias nulines hipotezes. </code></pre>
</section>
</section>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
